### PR TITLE
VTH-27: Section for Reporting Issues with the digital service

### PR DESCRIPTION
**Pull request by NavpreetGrewal**

### CODE REVIEW

File: fpo-api/api/views/FeedbackView.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import logging
+from rest_framework.request import Request
+from rest_framework.views import APIView
+from api.feedback import email_feedback
+from rest_framework.response import Response
+LOGGER = logging.getLogger(__name__)
+
+class FeedbackView(APIView):
```
Comment (WadeBarnes) [inline fpo-api/api/views/FeedbackView.py:8]: The feedback endpoint must be protected by reCAPTCHA, just like the form submission.
References:
- https://github.com/bcgov/violation-dispute-form/blob/master/fpo-api/api/views/SubmitTicketResponseView.py#L27
- https://github.com/bcgov/violation-dispute-form/blob/master/fpo-web/src/app/survey/survey.component.ts#L271
- https://github.com/bcgov/violation-dispute-form/blob/master/fpo-web/src/app/survey/survey.component.ts#L286

File: fpo-web/src/app/contact/contact.component.html [hunk 2]
```diff
@@ -1,3 +1,91 @@
...
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="reason" value="signup" id="reason_signup" required
+            [(ngModel)]="feedback.reason">
+          <label class="form-check-label" for="reason_signup">Find out how my government organization can use this</label>
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="reason" value="developer" id="reason_developer" required
+            [(ngModel)]="feedback.reason">
+          <label class="form-check-label" for="reason_developer">Explore clicky things as a software developer</label>
```
Comment (WadeBarnes) [inline fpo-web/src/app/contact/contact.component.html:55]: Remove the *Explore clicky things as a software developer* item.

File: fpo-api/api/views/FeedbackView.py (hunk 1 above)
Comment (NavpreetGrewal) [reply fpo-api/api/views/FeedbackView.py:12]: added reCAPTCHA verification method to feedbackView

File: fpo-web/src/app/contact/contact.component.html (hunk 2 above)
Comment (NavpreetGrewal) [reply fpo-web/src/app/contact/contact.component.html]: Done

**Pull request merged**